src/main/java/use_cases/calendar_scheduler/scheduler_use_case/SchedulerRequestModel.java

```diff
@@ -8,7 +8,6 @@
 public class SchedulerRequestModel {
 
     private Task task;
-    private ArrayList<Task> allTasks;
     private StudentUser user;
```

Comment:
If you want, you could get rid of the StudentUser parameter by using CurrentUser.getCurrentUser().